README.md

```diff
@@ -43,3 +43,10 @@ tabpfn.fit(X_train, y_train)
 tabpfn.predict(X_test)
 # or you can also use tabpfn.predict_proba(X_test)
 ```
+
+# Development
```

Comment:
Can you also mention this (https://docs.astral.sh/ruff/integrations/) to encourage developers to have the extensions installed in the editor?